tools/gui/qtimpfzentren.py

```diff
@@ -84,7 +84,7 @@ def get_horizontale_linie(self) -> QtWidgets.QFrame:
 
         return linie
 
-    def get_zentrum_widgets(self, gruppe: str, zentrum: dict) -> list[QtWidgets.QCheckBox, QtWidgets.QGridLayout]:
+    def get_zentrum_widgets(self, gruppe: str, zentrum: dict) -> list([QtWidgets.QCheckBox, QtWidgets.QGridLayout]):
```

Comment:
Werde dann aber noch einen neuen PR aufmachen um alles entsprechend zu ändern in den GUI - Dateien
Hier müsste man dann um es 3.8 kompatibel zu machen:
```
from typing import Tuple

..... -> Tuple[QtWidgets.QCheckBox, QtWidgets.QGridLayout]:
```